### ISSUE TITLE
Can't  Bio.pairwise2 align two very long sequences(5wbp and 10wbp)?

### ISSUE THREAD

**Issue by tanger-code** [biopython/biopython#4589]
Hi.
I am aligning two long sequence(seq1 is 5wbp, seq2 is 10wbp) using Bio.pairwise2, but there is an error:
`Process finished with exit code -1073741819 (0xC0000005)`

Can't Bio.pairwise2  align such long sequence? Or is there other method to do this in Bio?

Any advice would help me a lot.

**Comment by MarkusPiotrowski** [comment]
Sorry for my ignorance, but what is 'w' standing for?

**Comment by tanger-code** [comment]
> Sorry for my ignorance, but what is 'w' standing for?

Sorry, it means 50,000bp and 100,000bp

**Comment by MarkusPiotrowski** [comment]
Yes, that's probably too long, although I'm not sure if the error messages that you see is indicative for a memory overflow.

However, please use the 'new' PairwiseAligner, as described here: https://biopython.org/DIST/docs/tutorial/Tutorial.html#sec127

**Comment by peterjc** [comment]
I'd have written this as 50kbp and 100kbp, with k for kilo meaning a thousand from the SI units https://en.wikipedia.org/wiki/Metric_prefix

Was the "w" for 10,000 from another language perhaps? I know Japanese use this unit a lot.

**Comment by tanger-code** [comment]
> I'd have written this as 50kbp and 100kbp, with k for kilo meaning a thousand from the SI units https://en.wikipedia.org/wiki/Metric_prefix
> 
> Was the "w" for 10,000 from another language perhaps? I know Japanese use this unit a lot.

Haha, you guessed close, I'm Chinese.

**Comment by mdehoon** [comment]
@tanger-code  Were you able to use the `PairwiseAligner` for this problem? It will require a lot of memory (more than 50000 * 100000 bytes).

**Comment by mdehoon** [comment]
@tanger-code 

We are adding some convenience functions to `Bio.Align`, which similar functionality as the functions provided by `Bio.pairwise2`. If you have any comments or suggestions for these convenience functions, please let us know at issue #5290.
